node/release_autorun.sh

```diff
@@ -58,27 +47,49 @@ fetch() {
             new_release=true
         fi
     done
+    echo $new_release
 }
 
-fetch
-
-kill_process
+git_update_manager() {
+    while true; do
+        new_release=$(fetch)
+        git fetch
```

Comment:
I appreciate the work that went into this PR, but we cannot restore periodic git fetching, that is a firm condition on GitHub's willingness to restore this repository.